test.py

```diff
@@ -1,4 +1,3 @@
 from test1 import func
```

Comment:
Importing a function that's never used is unnecessary and decreases readability.

```suggestion
Remove the import statement if `func` is not intended to be used.
```